homework/kostyantyn-gamanyuk/06-lesson/build/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    
+    <title>06 Lesson (Shopping list)</title>
+    <link rel="stylesheet" href="/css/style.min.css">
+</head>
+<body>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <section class="shopping">
+    <div class="container">
+      <ul class="shopping__list">
+        <li>
+          <label>
```

Comment:
Label tag cannot be the parent tag for input, they must be used at the same level as in the example below
```
<div class="preference">
    <label for="cheese">Do you like cheese?</label>
    <input type="checkbox" name="cheese" id="cheese">
</div>
```